bahmnicore-api/src/main/java/org/bahmni/module/bahmnicore/service/impl/BahmniPatientServiceImpl.java

```diff
@@ -83,6 +89,23 @@ public PatientConfigResponse getConfig() {
                 searchParameters.getFilterPatientsByLocation(), searchParameters.getFilterOnAllIdentifiers());
     }
 
+    @Override
+    public List<PatientResponse> searchSimilarPatients(PatientSearchParameters searchParameters, PatientResponseMapper patientResponseMapper) {
```

Comment:
If we're going to use PatientResponseMapper in this service we should autowire it in the constructor, not ask for it to be passed in by the caller.